usersimcrs/simulator/neural_based/core/feature_handler.py

```diff
@@ -0,0 +1,23 @@
+"""Interface to build feature vector for neural-based user simulator."""
```

Comment:
Should we just call the package `neural` instead of `neural_based`?

Reply:
Good suggestion done.